src/metax_api/settings.py

```diff
@@ -199,6 +199,8 @@
 # endpoint in localhost where bearer tokens should be sent for validation
 VALIDATE_TOKEN_URL = 'https://127.0.0.1/secure/validate_token'
 
+CHECKSUM_ALGORITHMS = ['sha256', 'md5', 'sha512']
```

Comment:
Instead of defining the allowed algorithms in settings.py, the list should be read from app_config. The allowed algorithms could then be defined in metax-ops repo in roles/app_config/templates/app_config

Additionally, the list of values should be: SHA-256, SHA-512, MD5

Reply:
is it so that values are going to be EXACTLY like in the list. I made the list to be more flexible, because there are different ways to write algorithm names SHA-256/SHA256/sha256. Just checking :)

Reply:
Do not have permissions to push to metax-ops.